src/webots/gui/WbTcpServer.cpp

```diff
@@ -613,10 +613,11 @@ void WbTcpServer::sendWorldToClient(QWebSocket *client) {
 
   QString worlds;
   foreach (const QFileInfo item, worldFiles)
-    worlds += QDir(WbProject::current()->dir()).relativeFilePath(item.absoluteFilePath()) + ";";
+    worlds += QDir(WbProject::current()->dir()).relativeFilePath(item.absoluteFilePath()).split('/')[1] + ";";
```

Comment:
Why is that so complicated? Can we simply do it this way?
```suggestion
    worlds += item.fileName() + ";";
```